docs/tutorials/qunova-hivqe.ipynb

```diff
@@ -1121,7 +1122,9 @@
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
    "version": "3"
-  }
+  },
+  "platform": "cloud",
+  "title": "Compute Dissociation PES curve for Li2S (20Q) and FeP-NO (44Q) with Qunova HiVQE"
```

Comment:
Also a very long title

Reply:
We'll need to talk to the functions team (and the partner) to change this